cypress/integration/tests/developer/reports/filter.test.ts

```diff
@@ -156,20 +160,11 @@ describe("Reports filter validations", { tags: "@tier2" }, () => {
         // Check for current landscape donut charts
         cy.get("div > h2").eq(0).contains("No data available");
 
-        // Check for Adoption candidate distribution table
-        cy.get("div > h2").eq(1).contains("No data available");
-
-        // Check for Suggested adoption plan graphs
-        cy.get("div > h2").eq(2).contains("No data available");
-
-        // Check for Identified risks table
-        cy.get("div > h2").eq(3).contains("No results found");
-
         // Clear all filters
         clickByText(button, clearAllFilters);
     });
 
-    it("Description field validations", function () {
+    it.skip("Description field validations", function () {
```

Comment:
Why are we skipping this test ?

Reply:
https://issues.redhat.com/browse/TACKLE-696 We had this changes in 2.1, that's why skipped these tests as not relevant anymore now

Reply:
Ok if it is decided that these filters won't be there anymore then we need to remove these tests .